packages/cozy-ci-utils/README.md

```diff
@@ -0,0 +1,24 @@
+## cozy-ci-utils
+
+A series of utility commands for our CI.
+
+### Post a Github comment
+
+`cozy-ci-utils-github "But says hi"` Will post the message `Bot says hi` to the current PR.
+
+Requires an env var named `GH_TOKEN` which contains a Github token with the necessary permissions. It is recommended to generate this token from the @cozy-bot account.
```

Comment:
I would add a bit to precise that it only works on Travis.

Reply:
Meh, the first sentence of the readme says these scripts are for our CI, which is Travis — none of them will work outside of it.

Reply:
We are using Jenkins also.